fastlane_core/lib/fastlane_core/ui/errors/fastlane_error.rb

```diff
@@ -26,12 +26,12 @@ def could_contain_pii?
 
 class Exception
   def fastlane_crash_came_from_custom_action?
-    custom_frame = exception.backtrace.find { |frame| frame.start_with?('actions/') }
```

Comment:
Where does the `exception` object come from here though? Isn't it just `self` because it is within a monkeypatch?